### ISSUE TITLE
Search bar for extensions on front page

### ISSUE THREAD

**Issue by holly-cummins** [quarkusio/quarkusio.github.io#1656]
@maxandersen would like us to be able to search for extensions on the front page. This will be fairly easy to do once https://github.com/quarkusio/extensions/issues/43 is done.

**Comment by holly-cummins** [comment]
Oh hang on, this is a duplicate of #1595, which I also raised. :)
